test/resources/test_maskinporten.py

```diff
@@ -340,15 +339,7 @@ def test_create_client_key_to_aws(
         "key_password": None,
     }
 
-    maskinporten.ForeignAccountSecretsClient.send_secrets.assert_called_once_with(
-        ANY,
-        {
-            "key_id": ANY,
-            "keystore": ANY,
-            "key_alias": ANY,
-            "key_password": ANY,
-        },
-    )
+    maskinporten.ForeignAccountSecretsClient.send_secrets.assert_called_once()
```

Comment:
Det har ikke noe for seg å teste at vi faktisk sender de verdiene (hemmelighetene) vi skal lenger?